### DISCUSSION TITLE
avatar-scaling 内容美化

### DISCUSSION THREAD

**Discussion by slime-butter** [vrcd-community/vrchat-docs-localization#67] [General]
感觉这部分仍旧有可以美化的空间

## 译文
在 Udon 中，**目标视角球高度**最大高度限制是 100 米，最小高度限制是 0.1 米，而在模型菜单中，上下限制分别为 5 米和 0.2 米。通过上传超出高度限制的模型 ，且*不在其菜单里使用*模型缩放功能，您的模型将不存在高度限制。在这种情况下**目标视角球高度**与**预制体高度**一致，而后者高度不受限。

## 原文
The "Target Eye Height" is clamped between 0.1 and 100 meters for Udon, and the usable range of scaling in your Action Menu goes from 0.2 to 5.0 meters. But by uploading an avatar that is outside this range and not using the scale dial, you can still exceed those limits. In that case, "Target Eye Height" will match "Prefab Height," which is not limited.

## 链接
https://creators.vrchat.com/avatars/avatar-scaling

**Comment by SKPloft** [comment]
我认为这里也是，尤其是句首的“模型缩放”
## 译文
模型缩放将作为 **EyeHeightAsMeters** 参数自动同步给每个远端玩家，精度为 3 个小数点。而当远端玩家接收到新的**视角球高度**参数时，会在短时间内将对应模型平滑缩放到目标高度。
## 原文
Scale is automatically synchronized for each player as "eye height", quantized to 3 decimal points. For remote users, scale changes are smoothed over a short period any time a new height is received.
## 链接
同链接
